apps/wallet/src/ui/app/hooks/useGasBudgetEstimationUnits.ts

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { type CoinStruct } from '@mysten/sui.js';
+
+import { DEFAULT_GAS_BUDGET_FOR_PAY } from '_redux/slices/sui-objects/Coin';
+
+// This is a helper function to get a set of gas coins that can cover a given amount
+// It is from Coins.selectCoinSetWithCombinedBalanceGreaterThanOrEqual from sui.js
```

Comment:
should we make a backlog ticket and TODO to eventually consolidate this logic at the SDK level? it probably isn't good long-term to have two implementations floating around in case the logic ever changes